ncm-metaconfig/src/main/pan/components/metaconfig/schema.pan

```diff
@@ -86,6 +86,18 @@ type ${project.artifactId}_textrender_convert = {
 
 type caf_service_action = string with match(SELF, '^(restart|reload|stop_sleep_start)$');
 
+type ${project.artifactId}_actions = {
+    @{Always run, happens before possible modifications}
+    'pre' ? string
+    @{Always run, happens before possible modifications, the file content is passed om stdin.
+      Runs with 'keeps_state' enabled, so do not modify anything with this command.}
+    'test' ? string
+    @{Only run after file is modified, but after daemon action is added}
```

Comment:
What does "added" mean here? After the daemon action has been executed? Or just added to some queue for execution that will happen after these commands?

Reply:
it is always added to the queue (even if the changed command would fail), and then the changed command is executed. but the daemon action is only performed at the very end.

Reply:
```suggestion
    @{Only run after file is modified, but before any daemon action is executed. A failure in this command has no effect on whether the daemon action is executed later.}
```